packages/cli/src/utils/dependencies.js

```diff
@@ -0,0 +1,50 @@
+import consola from 'consola'
+import { satisfies } from 'semver'
+
+import { getPKG } from '@nuxt/utils'
+
+export function checkDependencies () {
+  const constraints = {
+    dependencies: {
+      postcss: {
+        range: '^7.0.32'
+      },
+      webpack: { range: '^4.46.0' }
+    },
+    optionalDependencies: {
+      'sass-loader': { range: '^10.1.1' }
+    },
+    engines: {
+      node: {
+        range: '>=12.0.0',
+        message:
+          'You are using an unsupported version of Node.js (x.y.z) and Nuxt is likely to be broken. It is recommended to use the latest LTS version (https://nodejs.org)'
```

Comment:
Add link to LTS release schedule.
```suggestion
          'You are using an unsupported version of Node.js (x.y.z) and Nuxt is likely to be broken. It is recommended to use the latest LTS version (https://nodejs.org/en/about/releases/)'
```